pegasus-build-env/ubuntu1804/Dockerfile

```diff
@@ -59,4 +59,13 @@ RUN add-apt-repository ppa:git-core/ppa -y; \
 
 RUN pip3 install --no-cache-dir cmake
 
+RUN wget http://dev-www.libreoffice.org/src/cppunit-1.13.2.tar.gz -P /opt/cppunit \
```

Comment:
We manage all the dependencies under https://github.com/XiaoMi/rdsn/blob/master/thirdparty/CMakeLists.txt and i think pegasus is still using gtest for unit-testing, not sure why you introduce cppunit here.